### DISCUSSION TITLE
Student certification application

### DISCUSSION THREAD

**Discussion by Buci-1** [community/community#51947] [GitHub Education]
I am a college student from China, majoring in computer, I keep trying to apply for student certification, and even submit application materials for the verbatim translation of student ID card and Xuexin.com certification, and more than 30 applications have failed. If none of this is successful, what else

**Comment by samus-aran** [comment]
Hi @Buci-1  👋
We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Comment by Buci-1** [comment]
I am very happy to receive an official reply. When I applied for student certification in the early days, my personal information was always incomplete, and I also looked up some information on the Internet, and also changed my personal information according to the information on the Internet, such as my personal name, the detailed address of the school, in short, I thought that the personal information I could provide had been provided. Including the certification of China's Xuexin network and student ID, since the information is Chinese, I also translate word by word according to the translation software, and the reason given above when the feedback information shows failure: Please consider using your device camera to submit academic affiliation documents. Uploaded images are easier to manipulate and therefore less trustworthy. So I also tried to get my student ID card information through the computer camera, but it still didn't work. So I tried to add a time watermark to the student ID card taken by my phone, but it still didn't pass. Below is a screenshot of my attached feedback.




---Original---
From: "Ashley ***@***.***&gt;
Date: Fri, Apr 7, 2023 05:13 AM
To: ***@***.***&gt;;
Cc: "Sun ***@***.******@***.***&gt;;
Subject: Re: [community/community] Student certification application(Discussion #51947)




 
Hi @Buci-1  👋
 We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Comment by ghost** [comment]
I graduated in China and I'm trying to get my transcript verification report and I keep receiving this text message from chsi.com:

【学信网】您好！ 您办理的大学成绩认证（西南石油大学-计算机科学与技术-本科）校对未通过，请登录认证网上申请平台，查看“我的消息”并按提示操作。

Translation: 【Xuexin.com hello!  Your university achievement certification (Southwest Petroleum University-Computer Science and Technology-Undergraduate) has not passed the proofreading. Please log in to the certification online application platform, check "My Message" and follow the prompts.


Can anyone please help me. I'd be very glad because my deadline is approaching.

**Reply by Buci-1** [reply]
您好，根据我不的不断尝试，你需要一份由学校盖章的文件，不论什么文件，带有学校盖章就行，可以是获奖证书
